### DISCUSSION TITLE
error message

### DISCUSSION THREAD

**Discussion by fredsevmartins** [Open-Systems-Pharmacology/Forum#809] [Questions & Problems]
Hi
I got this message, what could be?  it pup-up when I create a new simulation, and started after I open  2 OSP models  Itraconazole-Digoxin-DDI-master and  Rifampicin-Theophylline-DDI-master.

I already tried  : 1) decrease the solver tolerances of your simulation; 1 )increased the number of point to be simulated.
![PKSIM](https://user-images.githubusercontent.com/45317633/74819880-5b4bfa00-5301-11ea-9397-7e9f2e8f5b48.PNG)

Thanks for your time 
Frederico 

![PKSIM](https://user-images.githubusercontent.com/45317633/74818853-b11fa280-52ff-11ea-92d9-6c32f623c938.PNG)

**Comment by Yuri05** [comment]
please try [this](https://github.com/Open-Systems-Pharmacology/Forum/issues/328#issuecomment-494426826)

**Comment by fredsevmartins** [comment]
Hi @Yuri05 
I did, but didn't
![PKSIM](https://user-images.githubusercontent.com/45317633/74834527-df12e000-531b-11ea-8b91-455a1cdda248.PNG)
 work

**Comment by Yuri05** [comment]
as already mentioned in the [original discussion](https://github.com/Open-Systems-Pharmacology/Forum/issues/328#issuecomment-494426826): 
If you still get convergence issues - the **initial** model is with very high probability wrong.

In this case you can try to track the problem via Matlab ODE Export as descibed [here](https://github.com/Open-Systems-Pharmacology/Forum/issues/210#issuecomment-412834236)
If you don't haveMatlab: you can attach your project here, then we could investigate.

**Comment by msevestre** [comment]
@Yuri05 
>If you still get convergence issues - the initial model is with very high probability wrong.

How can this happen in PK-Sim? 

@fredsevmartins Can you describe a bit how your model was built? How is it different than usual PK-Sim models?

**Comment by fredsevmartins** [comment]
Hi @msevestre 
Every thing was doing well, till I open 2 models Itraconazole-Digoxin-DDI-master and Rifampicin-Theophylline-DDI-master from github library.
Now, all new simulation crash, even compound from library. Can I send you one of my simulation with caffeine ? How can I send to you?

Thanks 
Frederico

**Comment by StephanSchaller** [comment]
Hi Frederico,

just zip the PK-Sim file and drag and drop it here into the window when you write your message.

Best,
Stephan
